src/app/pages/task-calendar/rsync/rsync-form/rsync-form.component.ts

```diff
@@ -146,17 +150,18 @@ export class RsyncFormComponent {
         type: 'checkbox',
         name: 'rsync_archive',
         placeholder: T('Archive'),
-        tooltip: T('Equivalent to the <b>-rlptgoD</b> flag. This will\
-                    run the task as recursive, copy symlinks as symlinks,\
-                    preserve permissions, preserve modification times,\
-                    preserve group, preserve owner (root only), preserve\
-                    device files (root only), and preserve special files.'),
+        tooltip: T('Equivalent to passing the flags <b>-rlptgoD</b> to\
+                    rsync. When set, the rsync task is run as recursive,\
```

Comment:
When set, rsync is run recursively, preserving symlinks, permissions, modification times, group, and special files. When run as root, owner, device files, and special files are also preserved.

Reply:
updated. Left in the equivalent rsync flags.